app/models/concerns/wakes/metrics/facebook.rb

```diff
@@ -7,13 +7,26 @@ module Facebook
       included do
         store_accessor :document, :facebook_count, :facebook_count_updated_at
 
+        before_validation :prevent_update_of_legacy_location, on: :update
```

Comment:
no. my ellipsis was merely to indicate that I was moving my questioning to the `throw :abort` line